Sources/swift-format/Utilities/DiagnosticsEngine.swift

```diff
@@ -92,7 +91,7 @@ final class DiagnosticsEngine {
     for note in finding.notes {
       emit(
         Diagnostic(
-          severity: .note,
+          severity: .warning,
```

Comment:
Should this change be reverted? We're only removing severity from findings, but diagnostics should still support a `note` level severity.